tests/monotouch-test/ObjCRuntime/RegistrarTest.cs

```diff
@@ -75,6 +84,7 @@ public class RegistrarTest {
 			}
 		}
 
+#if !MONOMAC
```

Comment:
So the reason I #ifed these out is because they crash with a SIGSEGV.  In the Callproperty method these tests use, there's a call to:

`if (Runtime.Arch == Arch.DEVICE && expectFailure) {
  				Console.WriteLine ("Skipping '{0}', it's expected to throw a 'Selector not found exception', but on device it seems to crash instead", selector);
  				return;
  			}`

(which got commented out since mac doesn't have Runtime.Arch), so it looks like it crashses on device on iOS as well.  I could remove the #ifs to let the tests run but fix this call so it just returns instead of skipping completely, but then we just get a bunch of CWLs for the skipped tests and no extra useful data since we don't have a simulator to run it on like iOS does.